targets/switch/bmv2/Makefile

```diff
@@ -54,33 +54,20 @@ LIB_DIR := ${TARGET_ROOT}/build/
 MAKE_DIR := ${LIB_DIR}
 include ${MAKEFILES_DIR}/makedir.mk
 
-# Special p4 utils case:
-MODULE_BASE_DIR := $(P4FACTORY)
-
-MODULE := p4utils
-MODULE_LIB := $(LIB_DIR)/p4utils.a
-include $(MAKEFILES_DIR)/module.mk
-ifndef p4utils_OBJS_C
-  $(error No object files defined in p4utils_OBJS_C)
-endif
-$(p4utils_OBJS_C) : MODULE_INFO := p4utils
-
-
 SWITCHAPI_INC := ../../../submodules/switch/p4src/includes
 SWITCHAPI_INC += $(BMV2_PD_INC)
-SWITCHAPI_INC += $(P4FACTORY)/modules/p4utils/module/inc
 SWITCHAPI_INC += ${C_INCLUDE_PATH}
 SWITCHAPI_INC += ${TARGET_ROOT}
 SWITCHAPI_INC := $(subst $(space),:,$(SWITCHAPI_INC))
 
-SWITCHAPI_LIB := $(SUBMODULE_SWITCHAPI)/libswitchapi.a
+SWITCHAPI_LIB := $(SUBMODULE_SWITCHAPI)/.libs/libswitchapi.a
```

Comment:
Could not get it to compile w/o it. I think the makefile for tommyds is
setup to build libtommyds.la. When I tried to use library from switchapi
dir, toomy symbols stay unresolved.
Somewhere I saw that .libs are shared libs so I was wondering if it will
make the difference.. seem to work.

On Thu, Dec 10, 2015 at 3:10 PM, krambn <notifications@github.com> wrote:

> In targets/switch/bmv2/Makefile
> <https://github.com/p4lang/p4factory/pull/93#discussion_r47301110>:
>
> >  SWITCHAPI_INC += ${C_INCLUDE_PATH}
> >  SWITCHAPI_INC += ${TARGET_ROOT}
> >  SWITCHAPI_INC := $(subst $(space),:,$(SWITCHAPI_INC))
> >
> > -SWITCHAPI_LIB := $(SUBMODULE_SWITCHAPI)/libswitchapi.a
> > +SWITCHAPI_LIB := $(SUBMODULE_SWITCHAPI)/.libs/libswitchapi.a
>
> Do you want to use the .libs?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/p4lang/p4factory/pull/93/files#r47301110>.
>